erizo/src/erizo/MediaStream.cpp

```diff
@@ -152,6 +150,16 @@ boost::future<void> MediaStream::close() {
   });
 }
 
+void MediaStream::configure() {
```

Comment:
Why can't we do this in `MediaStream.init()`? I think semantically it's the same. For both it's things that we should do before using the media stream

Reply:
you're right, I've switched their names and now we only call init() from within MediaStream.cc